web/src/app/selection/QuerySelect.js

```diff
@@ -130,6 +130,8 @@ export const querySelectPropTypes = {
   multiple: p.bool,
   name: p.string,
   placeholder: p.string,
+
+  label: p.string,
```

Comment:
Is this needed? It looks like it's not being used within the file since it's bunched in with the `otherProps` spread